liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -641,6 +643,71 @@ private MavenProject getMavenProject(File buildFile) throws ProjectBuildingExcep
                         session.getProjectBuildingRequest().setResolveDependencies(true));
             return build.getProject();
         }
+        
+        @Override
+        protected void updateLooseApp() throws PluginExecutionException {
+        	// Check if we are using an exploded loose app
+        	if (LooseWarApplication.isExploded(project)) {
```

Comment:
This may be a dense question as I have not been keeping up with all of this, but what if the app is an `ear` application? Seems like we are assuming we are dealing with a `war` here, and that may be the case. Just wanted to clarify/understand.

Reply:
For now, we are just dealing with `war` applications. I believe the idea was to wait for a feature request for `ear` support.

Reply:
Is this method called for an ear application? If so, what happens?

Reply:
I dont believe so.... although I'm starting to question my understanding of the ear vs war flow :-).   This method is only called from ci.common code, so I dont think any ear/multi-module is at play. This PR also doesnt address multi-module yet, but am planning to address that as a next item.

Reply:
The fact it is in common does not affect what kind of application is being dealt with. I'm concerned about this `war` only approach. It seems like we at least need to know if it is ok to automatically call `LooseWarApplication` in `updateLooseApp` with no regard to what kind of app we have.

Reply:
I can add a check in this method to see what our packaging type is and only perform these actions if we are in fact a war app.